backend/pkg/api/handlers/public.go

```diff
@@ -95,8 +114,66 @@ func (h *HandlerService) PublicPutAccountDashboardTransactionsSettings(w http.Re
 	returnOk(w, nil)
 }
 
+// PublicPostValidatorDashboards godoc
+//
+//	@Description	Create a new validator dashboard. **Note**: New dashboards will automatically have a default group created.
+//	@Security 		ApiKeyInHeader || ApiKeyInQuery
+//	@Tags			Validator Dashboards
+//	@Accept			json
+//	@Produce		json
+//	@Param			request		body		handlers.PublicPostValidatorDashboards.request	true	"`name`: Specify the name of the dashboard.<br>`network`: Specify the network for the dashboard. Possible options are:<ul><li>`ethereum`</li><li>`gnosis`</li></ul>"
+//	@Success		201			{object}	types.PostValidatorDashboardsResponse
+//	@Failure		400			{object}	types.ApiErrorResponse
+//	@Failure		409			{object}	types.ApiErrorResponse	"Conflict. The request could not be performed by the server because the authenticated user has already reached their dashboard limit."
+//	@Router			/validator-dashboards [post]
```

Comment:
Would be cool if there was a way to even generate stubs of these. Like at least the route, return objects etc.

Reply:
best we can get is prob copilot auto completion 🤷